exp-player/addon/styles/components/exp-card-sort.scss

```diff
@@ -21,13 +21,19 @@
     font-size: 10px;
 }
 
+.small-bucket {
+    padding-right: 8px;
+    padding-left: 8px;
+}
+
 .small-bucket .well {
-    padding:12px;
+    padding:10px;
 }
 
 .small-bucket .alert {
     padding: 10px;
     margin-bottom: 12px;
+    overflow-wrap: break-word;
```

Comment:
Might want to check browser version. According to the MDN browser compatibility table for this property, support was added in... FF 49? Seriously?